includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -225,7 +225,6 @@ public static function register() {
 			'meta_box_cb'        => false, // See print_validation_errors_meta_box().
 			'capabilities'       => array(
 				'assign_terms' => 'do_not_allow',
-				'edit_terms'   => 'do_not_allow',
```

Comment:
In an earlier prototype I think the term edit page was being used, so to see it this would have had to be changed but we don't need it anymore. Good catch, I missed this while cleaning up.